data-model-for-portal/Microsoft.DocumentDB/cosmos-db.json

```diff
@@ -0,0 +1,296 @@
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az cosmosdb keys list",
+            "description": "List the access keys or connection strings for a Azure Cosmos DB database account.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/listKeys",
+            "confirmation": false,
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/cosmosdb/keys#az-cosmosdb-keys-list"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "--name",
+                            "--resource-group",
+                            "[--type]",
+                            "[--subscription]"
+                        ]
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "List the access keys or connection strings for a Azure Cosmos DB database account.",
+                    "parameters": [
+                        {
+                            "name": "--subscription",
+                            "value": "[path.subscriptionId]"
+                        },
+                        {
+                            "name": "--name",
+                            "value": "[path.accountName]"
+                        },
+                        {
+                            "name": "--resource-group-name",
+                            "value": "[path.resourceGroupName]"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az cosmosdb collection list",
+            "description": "Lists all Azure Cosmos DB collections.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/databases/{databaseRid}/collections",
+            "confirmation": false,
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/cosmosdb/collection#az-cosmosdb-collection-list"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "--db-name",
+                            "[--key]",
+                            "[--name]",
+                            "[--resource-group-name]",
+                            "[--url-connection]",
+                            "[--subscription]"
+                        ]
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "Lists all Azure Cosmos DB collections.",
+                    "parameters": [
+                        {
```

Comment:
`--db-name` is required for example.

Reply:
Thank you for your correction. Since there is no official example for some of the commands, I may have missed something when I created the example by myself, thank you very much for your help.